### DISCUSSION TITLE
Accessibility Soundcloud Widget on Elementor Pro Hello Theme

### DISCUSSION THREAD

**Discussion by Juliette2023** [soundcloud/api#274] [Q&A]
Hi, 

can someone please feedback on the soundcloud widget?

i am using Mac voiceover Screen reader on Mac OS 10.14.6  on desktop.  I'm in safari version 14.1.2 and chrome Version 116.0.5845.187 (Official Build) (x86_64). I am creating a website in in Elementor pro (hello theme). I know I am on quite an old version of Mac OS and thus also safari etc. But I cannot currently upgrade until i sort out my hard drive space.

1. The soundcloud widget seems to be keyboard operable in chrome and safari, although it doesn't have a focus indicator around each element which makes it difficult to see that it is selected, which is a problem for accessibility .  **Is there a  way i can add a focus indicator ?**

2. **Also is the soundcloud widget also accessible to screen readers ?**

If i go through using mac voiceover with the tab keys the play button seems to be indicated correctly - VoiceOver says "play, web application' so to some degree indicates what the user has to do.

However if i go through VoiceOver using the mac recommended voiceover operation keys (control, option and right arrow key) the below happens:

safari - When it reaches the Soundcloud widget - It says it is a frame and i can enter the frame, so I do that.  Then the first thing it says "In frame 0' and I have to tab on to the next link and come back and it then says 'play, web application'. This is very unclear indeed what is going on as the user does not know to play the button. 

chrome - In  it says 'soundcloud widget frame'  and that I can enter it. I do that and it says 'you are currently on an image' and then I tab on and it remains in the same place and tab and says 'end of, play, application'. This is clearer as at least it says this when I land on it, but still not that clear what is going on.

**Comment by dpreussler** [comment]
can you open this as an issue?

**Reply by Juliette2023** [reply]
How do I do this?

**Reply by dpreussler** [reply]
just post it here: https://github.com/soundcloud/api/issues
